practice_points.c

```diff
@@ -0,0 +1,107 @@
+#include <cstring>
```

Comment:
Replace C++ version with C one.
```suggestion
#include <string.h>
```